src/components/search/SearchCourseCard.jsx

```diff
@@ -11,9 +11,10 @@ import { sendEnterpriseTrackEvent } from '@edx/frontend-enterprise-utils';
 
 import { getPrimaryPartnerLogo, isDefinedAndNotNull } from '../../utils/common';
 
-const SearchCourseCard = ({ hit, isLoading }) => {
+function SearchCourseCard({ hit, isLoading }) {
   const { enterpriseConfig: { slug, uuid } } = useContext(AppContext);
 
+  // eslint-disable-next-line react-hooks/exhaustive-deps
   const course = hit ? camelCaseObject(hit) : {};
```

Comment:
This is a valid ESLint error we shouldn't ignore. On every re-render of `SearchCourseCard`, the `course` variable be have a new reference, which will make `linkToCourse` re-compute on every re-render as well. Should we compute `course` with a `useMemo`?